File: Unidad 2-Estructuras de Control/Ejercicio19.js
/*
Ejercicio 19 Javasricript
Unidad 2 - {Programación Estructurada}
Equipo 1
Autor: Daniel Eduardo Gutierrez Delfin
Fecha: 8 de Febrero 2019
Entradas: 3 numeros
Salidas: El numero mayor
---------------------------------------
Instrucción:
Escribir un programa que lea tres números y determine el mayor de los tres.
---------------------------------------
Procedimiento General:
---------------------------------------
1.Se leen los 3 numeros dados por el usuario.
2.Se verifica que los numeros no sean iguales
3.Vemos si el primer numero es mayor que el segundo, si es asi vemos si es mayor al tercero y si es asi se imprime el primer numero
4.Si el primer numero no es mayor al segundo, vemos si el segundo numero es mayor al tercero y si es asi se imprime el segudno numero
5.Si no es ninguno de los primeros 2 cosos por default el mayor es el tercer numero y se imprime.
---------------------------------------
*/

//entradas//
let numero1 = parseFloat(prompt('Ingresa el número 1'));
let numero2 = parseFloat(prompt('Ingresa el número 2'));
let numero3 = parseFloat(prompt('Ingresa el número 3'));

let numeroMayor = null;

//proceso//
if (numero1>numero2){//vemos si el primer numero es mayor al segundo//
  if (numero1>numero3){//Si es asi vemos si el rpimer numero es mayor al tercero//
    numeroMayor = numero1; //si es asi el numero 1 se guarda en la variable//
  }
  else{
    numeroMayor = numero3;// si no el numero 3 toma el valor de la variable//
  }
} else {
  if (numero2 > numero3){//si el primer numero no fue mayor al segundo vemos si el numero 2 es mayor que el numero 3//
    numeroMayor = numero2;//si es asi el numero 2 se guarda en la variable//
  } else if (numero3 > numero2){ //si no es asi se rectifica en inversa//
    numeroMayor = numero3;//y si es verdad el tercer numero se guarda en la variable//
  }
}

//Salidas//
if(numeroMayor){
  alert("El número mayor es" + numeroMayor);//se imprime la varible con el numero guardado//
}else {
  alert("Los números son iguales");//si no se cumplio ningun caso los numeros son iguales//
}
  

/*Autor QA:Emmanuel Azcorra Balam
Entradas:(1,2,3),(3,2,1),(2,1,3),(9,3,10)
salidas:3,3,3,10
Proceso :Ok, todo correcto.*/

File: Unidad 2-Estructuras de Control/Ejercicio19.py
#Ejercicio 19 
#Escribir un programa que lea tres números y determine el mayor de los tres.
#Autor: Pamela Citlali Canul Chacón (Equipo 'about:blank' )

#Dato de entrada: Tres números
#Dato de salida: El número mayor entre los tres.

#Entrada
#Definimos una variable donde se guardará al número mayor
numMayor = 0 

#Se definen las variables a utilizar
num1 = 0
num2 = 0 
num3 = 0 

#Se solicita que el usuario ingrese los números 
num1 = float(input("Ingresa un numero: "))
num2 = float(input("Ingresa un segundo numero: "))
num3 = float(input("Ingresa un tercer numero: "))

#Procedimiento 
#Primero se analiza si el número 1 es el mayor, comporándolo con los otros números
if num1 > num2 and num1 > num3:
    #Si resulta el mayor de los tres se guarda en la variable antes definida
    numMayor = num1 
#Si el primer número no es el mayor se analiza el segundo número
elif num2 > num1 and num2 > num3:
    #Si resulta el mayor de los tres se guarda en la variable antes definida
    numMayor = num2
#Si nunguno de los dos primeros números resulta el mayos, entonces, se infiere que el tercer número es el mayor
else: 
    #Se guarda el valor en la variable antes definida
    numMayor = num3

#Salida
#Por último se imprime el resultado 
print ("El número mayor es:" , numMayor)